CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ## Added
 - Util to compare value types
 - "If...Then...Else": util file to display human format formula
+- Store: add two new mutations `setAvailableVariables` and `setVariablesDelimiters` to populate state properties `availableVariables` and `variablesDelimiters`
```

Comment:
I think it's not necessary to add this in the functional changelog.
It will be more relevant to write here the external behavior of weaverbird will change, like when we will be using these properties in some form widgets, and when it will become necessary for weaverbird integrators to use this store mutations to make variables available there.